src/3rd_party_repos.c

```diff
@@ -543,9 +543,13 @@ enum swupd_code third_party_run_operation_multirepo(const char *repo, process_bu
 void third_party_repo_header(const char *repo_name)
 {
 	char *header = NULL;
+	bool quiet = (log_get_level() == LOG_ERROR);
```

Comment:
This is hard to maintain.
I would create a `#define LOG_QUIET LOG_ERROR` in log.h or some kind of function log_is_debug() and log_is_quiet().
And the ideal would be `log_get_level() < LOG_ERROR` to include LOG_ALWAYS too.

Reply:
good idea